src/Pages/Login-Page/Login.jsx

```diff
@@ -0,0 +1,82 @@
+import { Nav } from "../../components/Nav/Nav";
+import { Header } from "../../components/Header/Header";
+import { Footer } from "../../components/Footer/Footer";
+import { NavForTab } from "../../components/NavForTab/NavForTab";
+import { NavLink } from "react-router-dom";
+import "./login.css";
+import "./loginResponsive.css";
+export function Login() {
+  return (
+    <div className="login-page">
+      <Nav />
+      <Header />
```

Comment:
Rest all things are clean and properly formatted

Reply:
ThankYou